### PR TITLE
Update artifacts for ci.yml

### PR DESCRIPTION
**Pull request by gearama**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-administration/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,3 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# SPDX-License-Identifier: MIT
-
```
Comment (ahsonkhan) [inline sdk/keyvault/azure-security-keyvault-administration/vcpkg.json:3]: 👍

File: sdk/keyvault/ci.yml [hunk 2]
```diff
@@ -41,6 +41,9 @@ stages:
         - Name: azure-security-keyvault-certificates
           Path: azure-security-keyvault-certificates
           VcpkgPortName: azure-security-keyvault-certificates-cpp
+        - Name: azure-security-keyvault-administration
+          Path: azure-security-keyvault-administration
+          VcpkgPortName: azure-security-keyvault-administration-cpp
```
Comment (ahsonkhan) [inline sdk/keyvault/ci.yml:46]: I am not as familiar with keyvault CI yaml. Are these two the only changes needed to light up CI runs?

Are nightly/internal pipelines running?

**Review by ahsonkhan — APPROVED** [review]
Looks reasonable to me, but I recommend getting someone on EngSys to validate as well. cc @danieljurek

File: sdk/keyvault/ci.yml (hunk 2 above)
Comment (gearama) [reply sdk/keyvault/ci.yml:46]: tey are running , it's about publishing artifacts in the piplines.

File: cmake-modules/FolderList.cmake [hunk 3]
```diff
@@ -12,6 +12,9 @@ macro(GetFolderList project)
     elseif(${project} STREQUAL KEYS)
         DownloadDepVersion(sdk/core azure-core 1.5.0)
         DownloadDepVersion(sdk/identity azure-identity 1.1.0)
+    elseif(${project} STREQUAL ADMINISTRATION)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
```
Comment (LarryOsterman) [inline cmake-modules/FolderList.cmake:16]: Have you actually tested building keyvault-administration against azure-core 1.5.0? Should this instead be 1.8?

File: sdk/keyvault/azure-security-keyvault-administration/vcpkg.json [hunk 4]
```diff
@@ -1,6 +1,3 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# SPDX-License-Identifier: MIT
```
Comment (LarryOsterman) [inline sdk/keyvault/azure-security-keyvault-administration/vcpkg.json:2]: IIRC our vcpkg publishing pipeline strips these off, but in general we keep them.

**Review by LarryOsterman — CHANGES_REQUESTED** [review]

File: sdk/keyvault/azure-security-keyvault-administration/vcpkg.json (hunk 4 above)
Comment (ahsonkhan) [reply sdk/keyvault/azure-security-keyvault-administration/vcpkg.json:2]: From what I can tell (and FWIW), that's for all the `vcpkg/vcpkg.json` files (the vcpkg.json file nested in the vcpkg/ folder for each service). The other "root" vckpg.json files don't have this comment.

Comment (gearama) [reply sdk/keyvault/azure-security-keyvault-administration/vcpkg.json:2]: No we do not keep these comments, check the other folders , this is the root vcpkg.json for the package not the one in the vcpkg folder.

Comment (gearama) [reply sdk/keyvault/azure-security-keyvault-administration/vcpkg.json:2]: even the attestation one is missing thhe header, the build actually fails if the comment is there vcpkg complkains that it is invalid json

File: cmake-modules/FolderList.cmake (hunk 3 above)
Comment (gearama) [reply cmake-modules/FolderList.cmake:16]: yes

**Comment by gearama** [comment]
@LarryOsterman  you are blocking my merge,

**Review by antkmsft — APPROVED** [review]
LGTM. @LarryOsterman, the changes are right; it is confusing that we have two instances of the `vcpkg.json` file - one is a template for vcpkg releases, the other one is basically a part of the development-time project system, and it is needed for enabling using vcpkg in manifest mode during development.
What happened here, is that George mistakenly copy-pasted the template file to be the manifest mode file as well.
I have a change in mind that will make it easier to distinguish between them.

**Review by LarryOsterman — APPROVED** [review]

**Review by danieljurek — COMMENTED** [review]
LGTM

**Pull request merged**